src/components/modules/blockSelection.ts

```diff
@@ -175,7 +175,7 @@ export default class BlockSelection extends Module {
          *
          * Prevent such actions if focus is not inside the Editor
          */
-        if (!BlockManager.currentBlock) {
+        if (!event.target.closest(`.${this.Editor.UI.CSS.editorWrapper}`)?.isEqualNode(this.Editor.UI.nodes.wrapper)) {
```

Comment:
When multiple Editor.js instances are launched, `BlockManager.currentBlock` can't be changed to null by focusing other instances.

Reply:
maybe it makes sense to create a function inside UI module that gets event and checks if event's target is under the current editors instance wrapper?

Reply:
Exactly, this method seems DOM hacking!
Sorry, this problem has been resolved by https://github.com/codex-team/editor.js/pull/1431